scapy/contrib/isotp.uts

```diff
@@ -737,9 +738,9 @@ with ISOTPSoftSocket(cans, sid=0x641, did=0x241) as s:
             raise ex
     thread = threading.Thread(target=sender, name="sender")
     thread.start()
-    ready.wait()
+    ready.wait(15)
```

Comment:
Is there a need to configure this value?

Reply:
I don't think so. This is just to avoid Travis freeze, and rather have the error displayed.